server/server_test.go

```diff
@@ -131,9 +131,9 @@ func TestGetHandler(t *testing.T) {
 
 			meta, err := server.HandleGet(rec, req)
 			if tt.expectError {
-				require.Error(t, err)
+				require.Error(t, err.Err)
```

Comment:
Do you need to unwrap the err.Err like this? The whole point of custom errors is that `err` itself is an error, you shouldn't need to unwrap it I would think?

Reply:
Thanks for pointing this out. I needed to unwrap with HandlerError defined with `Meta *commitments.CommitmentMeta`, but updating to `Meta commitments.CommitmentMeta` allowed me to directly access it like an error.

Reply:
oh interesting... and weird! But glad this works now